lib/middleware/auth.js

```diff
@@ -58,14 +57,16 @@ exports.initialize = function initialize () {
         }
 
         /**
-         * we need to generate the url
-         * and then request it
+         * Performing a request to the brf auth endpoint creates an account if it does not exist.
+         * We actually don't care about the response.
          */
 
-        var authPath = process.env.FORUM_URL + '/auth/brf?brfauth=' + nodebb.authJWT(user)
-        request(authPath, function (error, response, body) {
-          if (error) console.log('error:', error) // Print the error if one occurred
-        })
+        const authPath = process.env.FORUM_URL + '/auth/brf?brfauth=' + nodebb.authenticationToken(user)
+        try {
+          await got(authPath)
+        } catch (e) {
+          console.log('Error while touching user forum account: ' + e);
```

Comment:
It would be excellent if we'd log these kinds of errors. Makes it easier with debugging down the line. See how it's done elsewhere: https://github.com/CIVIS-project/BRFApp/blob/89c1b9dde8b373d8bb1b2eb79ba342ba01c518ed/lib/routes/auth.js#L91-L95